csrc/vmm/torch_bindings.cpp

```diff
@@ -0,0 +1,27 @@
+#include "registration.h"
+#include "vmm.h"
+
+TORCH_LIBRARY_EXPAND(TORCH_EXTENSION_NAME, m) {
+  // CacheDevicePtr class bind
+  m.class_<CacheDevicePtr>("CacheDevicePtr")
+      .def(torch::init<>())
+      // .def_readwrite("dptr", &CacheDevicePtr::dptr);
+      // dptr cann't bind success because of the type of dptr is
+      // CUdeviceptr(=unsigned long long), which is not supported in torch
```

Comment:
Are you going to resolve this?

Reply:
No use for .dptr in python at the moment, I'll delete here